packages/settings-library/tests/test_utils_r_clone.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=redefined-outer-name
+
+import pytest
+from settings_library.r_clone import RCloneSettings, S3Provider
+from settings_library.utils_r_clone import _COMMON_ENTRIES, get_r_clone_config
+
+
+@pytest.fixture(params=list(S3Provider))
+def r_clone_settings(request, monkeypatch) -> RCloneSettings:
+    monkeypatch.setenv("R_CLONE_PROVIDER", request.param)
+    monkeypatch.setenv("R_CLONE_STORAGE_ENDPOINT", "storage_endpoint")
+    monkeypatch.setenv("S3_ENDPOINT", "endpoint")
+    monkeypatch.setenv("S3_ACCESS_KEY", "access_key")
+    monkeypatch.setenv("S3_SECRET_KEY", "secret_key")
+    monkeypatch.setenv("S3_BUCKET_NAME", "bucket_name")
+    monkeypatch.setenv("S3_SECURE", False)
```

Comment:
Maybe add the aws-region everywhere as a configurable variable as well (vide supra)